lib/commands/init.js

```diff
@@ -126,10 +124,6 @@ module.exports = Command.extend({
     let blueprintOpts = clone(commandOptions);
     let blueprint = normalizeBlueprint(blueprintOpts.blueprint || this._defaultBlueprint());
 
-    if (isExperimentEnabled('VITE') && blueprint === 'app') {
```

Comment:
Is this change in pattern an issue for ember-cli-update using the blueprint from config?